### ISSUE TITLE
ARCore continuous image tracking

### ISSUE THREAD

**Issue by juice49** [viromedia/viro#828]
Hi!

Am I correct in my understanding that Viro does not currently support continuous image tracking in ARCore?

All documentation suggests this is the case, however @achuvm notes in #577 that ARCore was upgraded to 1.10.0 with continuous image tracking in Viro React 2.15.0.

It would be very useful to know for certain whether Viro now supports ARCore continuous image tracking. Thank you for any help 🙂.

**Comment by rachitgupta98** [comment]
Yeah its true now you can implement continuous image tracking for Android also.
